platform/view/view/context.go

```diff
@@ -72,6 +76,12 @@ type MutableContext interface {
 
 // Context gives a view information about the environment in which it is in execution
 type Context interface {
+	// StartSpanFrom creates a new child span from the passed context
+	StartSpanFrom(ctx context.Context, name string, opts ...trace.SpanStartOption) (context.Context, trace.Span)
```

Comment:
can we move these two functions to a different interface that is imported by Context?